modules/apps/site-initializer/site-initializer-raylife-d2c/test.properties

```diff
@@ -23,7 +23,7 @@
     #
 
     test.batch.run.property.query[functional-tomcat*-mysql*-jdk8][relevant]=\
-        (portal.acceptance == true) AND \
+        (portal.acceptance == talitha) AND \
```

Comment:
Hi @itstali
This value of portal.upstream can't be merged. Would you please change back to "true"?